src/mono/netcore/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -9,6 +9,7 @@
     <GenerateNeutralResourcesLanguageAttribute>false</GenerateNeutralResourcesLanguageAttribute>
     <EnsureRuntimePackageDependencies>false</EnsureRuntimePackageDependencies>
     <TargetFramework>netcoreapp2.1</TargetFramework>
+    <EnablePInvokeAnalyzer>false</EnablePInvokeAnalyzer>
```

Comment:
should this be reenabled at some point? if yes we should add a TODO or file an issue.

Reply:
There is already ##33867